pkg/conversion/decode.go

```diff
@@ -59,7 +72,9 @@ func (s *Scheme) Decode(data []byte) (interface{}, error) {
 // technique. The object will be converted, if necessary, into the versioned
 // type before being returned. Decode will not decode objects without version
 // set unless version is also "".
-func (s *Scheme) DecodeToVersion(data []byte, version string) (interface{}, error) {
+// a GroupVersion with .IsEmpty() == true is means "use the internal version
+// for this group
```

Comment:
https://github.com/kubernetes/kubernetes/pull/17432/files#diff-ded85789c18cc8c3fc7b4038b75423a8R51 won't return true for any group other than "", right?